packet/__init__.py

```diff
@@ -27,11 +28,11 @@
 db = SQLAlchemy(app)
 migrate = Migrate(app, db)
 
-auth = OIDCAuthentication(app, issuer=app.config["OIDC_ISSUER"], client_registration_info={
-    "client_id": app.config["OIDC_CLIENT_ID"],
-    "client_secret": app.config["OIDC_CLIENT_SECRET"],
-    "post_logout_redirect_uris": "/logout/"
-})
+APP_CONFIG = ProviderConfiguration(issuer=app.config["OIDC_ISSUER"],
```

Comment:
`issuer=` can be omitted because it's the first argument of the `ProviderConfiguration` constructor, but this is still fine.